# MINCO trajectory

time start_time
geometry_msgs/Point[] pos_pts
geometry_msgs/Point[] angle_pts
geometry_msgs/Vector3 init_v
geometry_msgs/Vector3 init_a
float64[] posT_pts
float64[] angleT_pts
